packages/dds/tree/src/tree/pathTree.ts

```diff
@@ -87,3 +87,13 @@ export function clonePath(path: UpPath | undefined): UpPath | undefined {
         parentIndex: path.parentIndex,
     };
 }
+
+export function topDownPath(path: UpPath | undefined): UpPath[] {
```

Comment:
Reusing the `UpPath` type for the elements in the "down path" seems misleading.

Reply:
They are still valid `UpPath` elements (they don't get modified at all). I added a comment to help clarify.